### ISSUE TITLE
Extra calls to /v1/graphs that appear unnessessary

### ISSUE THREAD

**Issue by grechaw** [marklogic/marklogic-jena#52]
Looking at the access logs for stress tests, there are periodic calls to /v1/graphs that get a 404 for no default graph.  It seems that this call should be tracked down and removed.

```
" 204 0 - -
172.18.128.184 - admin [03/Oct/2016:13:04:56 -0700] "POST /v1/graphs/sparql HTTP/1.1" 204 0 - -
172.18.128.184 - admin [03/Oct/2016:13:04:56 -0700] "POST /v1/graphs/sparql HTTP/1.1" 204 0 - -
172.18.128.184 - admin [03/Oct/2016:13:04:57 -0700] "POST /v1/graphs/sparql?base=file:///space/stress/sql/ HTTP/1.1" 200 657 - -
172.18.128.184 - admin [03/Oct/2016:13:04:57 -0700] "GET /v1/graphs?graph=http://marklogic.com/semantics%23default-graph HTTP/1.1" 404 269 - -
172.18.128.184 - admin [03/Oct/2016:13:04:57 -0700] "POST /v1/graphs/sparql?base=file:///space/stress/sql/ HTTP/1.1" 200 664 - -
172.18.128.184 - admin [03/Oct/2016:13:04:57 -0700] "GET /v1/graphs?graph=http://marklogic.com/semantics%23default-graph HTTP/1.1" 404 269 - -
172.18.128.184 - admin [03/Oct/2016:13:04:57 -0700] "POST /v1/graphs/sparql HTTP/1.1" 204 0 - -
172.18.128.184 - admin [03/Oct/2016:13:04:57 -0700] "POST /v1/graphs/sparql HTTP/1.1" 204 0 - -
```

I'm assuming this is a marklogic-jena bug.

**Comment by grechaw** [comment]
Put 1.0.2 milestone on for verification.  Not sure whether it will make that release, but we'd like to understand it better sooner.

**Comment by grechaw** [comment]
Potentially big perf impact, so good to nail down if possible

**Comment by grechaw** [comment]
This bug was caused by the test harness's using a version older than the fixes in #29
